src/schema/v2/artwork/index.ts

```diff
@@ -358,6 +358,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           existyValue(collecting_institution),
       },
       comparableAuctionResults: ComparableAuctionResults,
+      confidentialNotes: {
+        type: GraphQLString,
+        resolve: ({ confidential_notes }) => existyValue(confidential_notes),
```

Comment:
**Question**: what is special about this field to require `existyValue` to be used? Never came across this usage before so would like to understand this. Is it that the field may come as an empty object or empty string and if so, don't even set it? Thanks!

Reply:
🙈 I copied this because I saw the field above using it, I don't think it is necessary the other fields just return null when they are not present without this which I think is fine, I will remove it